packages/core/test/unit/Site.test.js

```diff
@@ -276,77 +276,85 @@ test('Site should not deploy without a built site', async () => {
         + 'Please build the site first before deploy.'));
 });
 
-describe('Site deploy with Travis', () => {
-  // Keep a copy of the original environment as we need to modify it for deploy Travis tests
-  const OLD_ENV = { ...process.env };
-
-  beforeEach(() => {
-    // Delete all environment variables that affect tests
-    delete process.env.TRAVIS;
-    delete process.env.GITHUB_TOKEN;
-    delete process.env.TRAVIS_REPO_SLUG;
```

Comment:
Let's keep the old approach of deleting / resetting (`afterAll`) the environment variables, to ensure our tests run independent of the CI environment

Reply:
Thanks for catching this. I have added them back.